### DISCUSSION TITLE
Themed version of webviews

### DISCUSSION THREAD

**Discussion by cyco77** [Power-Maverick/DataverseDevTools-VSCode#117] [General]
I just started development of theme enabled version of webviews:
- Entity List
- Entity Details
- Smart Match
- ... 

I tried to use VSCode UI Toolkit and I think it looks not so bad :)

![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/919a0ba7-31e2-4c97-a0f9-02f908c7a502)
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/ee948906-07c5-45f8-9b31-a570705547dc)

Do you have additional ideas regarding styling and and layout?

Best regards,
Lars

**Comment by Power-Maverick** [comment]
The headers need to standout from the rest of the rows; otherwise, it looks beautiful.

**Comment by Power-Maverick** [comment]
Did you ever create a PR for this change?

**Comment by cyco77** [comment]
sry... a long time ago... I had to many other topics on my desk the last couple of month... will try to continue this topic soon

**Comment by Power-Maverick** [comment]
Waiting for the PR as this will improve the tool alot. :)

**Comment by cyco77** [comment]
now I completed first themed version... please take a look at the pr https://github.com/Power-Maverick/DataverseDevTools-VSCode/pull/233

**Comment by cyco77** [comment]
had some fun with my old repo... updated everything and published new PR :)
